api/app/src/TestHelpers/DeputyTestHelper.php

```diff
@@ -6,11 +6,12 @@
 
 use App\Entity\Deputy;
 use App\Entity\User;
+use Doctrine\ORM\EntityManager;
 use Faker\Factory;
 
 class DeputyTestHelper
 {
-    public static function generateDeputy(?string $email = null, ?string $deputyUid = null, ?User $user = null): Deputy
+    public static function generateDeputy(?string $email = null, ?string $deputyUid = null, ?User $user = null, ?EntityManager $em = null): Deputy
```

Comment:
Looking at this in more detail, I would just get the returned Deputy and save it, rather than passing an entity manager around.